src/test/java/model/process/SortProcessTest.java

```diff
@@ -0,0 +1,214 @@
+		DataProcess process = new SortProcess(new RowValueDescriber<>(boolcol), asc);
+		DataTable result = proc(process);
+		assertEquals(new StringValue("abc"), result.getRow(0).getValue(stringcol));
+	}
+	
+	@Test
+	public void testSortingFloat() throws Exception {
+		DataProcess process = new SortProcess(new RowValueDescriber<>(floatcol), asc);
+		DataTable result = proc(process);
+		assertEquals(new StringValue("opq"), result.getRow(0).getValue(stringcol));
+	}
+	
+	@Test
+	public void testSortingInt() throws Exception {
+		DataProcess process = new SortProcess(new RowValueDescriber<>(intcol), asc);
+		DataTable result = proc(process);
+		assertEquals(new StringValue("abc"), result.getRow(0).getValue(stringcol));
+	}
+	
+	/**
+	 * REVERSE, REVERSE!
+	 * https://youtu.be/wZv62ShoStY?t=3m17s
+	 */
+	
+	@Test
+	public void testReverseSortingDateTime() throws Exception {
+		DataProcess process = new SortProcess(new RowValueDescriber<>(datetimecol), desc);
+		DataTable result = proc(process);
+		assertEquals(new StringValue("klm"), result.getRow(0).getValue(stringcol));
+	}
+	
+	@Test
+	public void testReverseSortingDate() throws Exception {
+		DataProcess process = new SortProcess(new RowValueDescriber<>(datecol), asc);
+		DataTable result = proc(process);
+		assertEquals(new StringValue("abc"), result.getRow(0).getValue(stringcol));
+	}
+	
+	@Test
+	public void testReverseSortingTime() throws Exception {
+		DataProcess process = new SortProcess(new RowValueDescriber<>(timecol), asc);
+		DataTable result = proc(process);
+		assertEquals(new StringValue("abc"), result.getRow(0).getValue(stringcol));
+	}
+	
+	@Test
+	public void testReverseSortingPeriod() throws Exception {
+		DataProcess process = new SortProcess(new RowValueDescriber<>(periodcol), desc);
+		DataTable result = proc(process);
+		assertEquals(new StringValue("opq"), result.getRow(0).getValue(stringcol));
+	}
+	
+	@Test
+	public void testReverseSortingString() throws Exception {
+		DataProcess process = new SortProcess(new RowValueDescriber<>(stringcol), desc);
+		DataTable result = proc(process);
+		assertEquals(new StringValue("abb"), result.getRow(0).getValue(stringcol));
+	}
+	
+	@Test
+	public void testReverseSortingBool() throws Exception {
+		DataProcess process = new SortProcess(new RowValueDescriber<>(boolcol), desc);
+		DataTable result = proc(process);
+		assertEquals(new StringValue("abc"), result.getRow(0).getValue(stringcol));
+	}
+	
+	@Test
+	public void testReverseSortingFloat() throws Exception {
+		DataProcess process = new SortProcess(new RowValueDescriber<>(floatcol), desc);
+		DataTable result = proc(process);
+		assertEquals(new StringValue("opq"), result.getRow(0).getValue(stringcol));
+	}
+	
+	@Test
+	public void testReverseSortingInt() throws Exception {
+		DataProcess process = new SortProcess(new RowValueDescriber<>(intcol), desc);
+		DataTable result = proc(process);
+		assertEquals(new StringValue("abc"), result.getRow(0).getValue(stringcol));
```

Comment:
Je resultaten in de reverse tests zijn hetzelfde als in de gewone tests, blijkbaar gaat er toch ergens iets fout.